ddtrace/ext/ci.py

```diff
@@ -207,6 +210,16 @@ def extract_azure_pipelines(env):
         git.COMMIT_AUTHOR_EMAIL: env.get("BUILD_REQUESTEDFOREMAIL"),
         STAGE_NAME: env.get("SYSTEM_STAGEDISPLAYNAME"),
         JOB_NAME: env.get("SYSTEM_JOBDISPLAYNAME"),
+        CI_ENV_VARS: json.dumps(
+            OrderedDict(
```

Comment:
the `OrderedDict` is so that we can safely assert against a fixed fixture. It's a bit awkward but couldn't come up with something better. Open to suggestions

Reply:
ah, yeah. The only other option I can think of is to parse the fixture and parse the generated data, and then compare.

but you are asserting that `json.dumps(dict()) == json.dumps(dict())` right?

Reply:
while weird that we are using an `OrderedDict` when order does not actually matter is weird, I can understand why we are doing it here.

Can you leave a comment on each usage of `OrderedDict` stating that it is because of testing, etc.

I want to make sure someone doesn't come through and try to remove it later.

Reply:
for sure 👍